compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -1340,6 +1340,34 @@ bool DerivedThreadConfigAttr::hasTilingLevel(unsigned level) const {
   return level == llvm::to_underlying(GPU::TilingLevel::Thread);
 }
 
+//===----------------------------------------------------------------------===//
+// UseGlobalLoadDMAAttr
+//===----------------------------------------------------------------------===//
+
+SmallVector<int64_t>
+UseGlobalLoadDMAAttr::getStaticTilingLevelSizes(unsigned level,
```

Comment:
Why's there an `unsigned` in here?

Reply:
This is how the LoweringConfigAttrInterface exposes tiling levels. It's up to the backend + lowering config to interpret the level consistently.